QueryManager/src/main/java/eu/neclab/ngsildbroker/queryhandler/repository/QueryDAO.java

```diff
@@ -77,6 +77,10 @@ public Uni<Map<String, Object>> getEntity(String entityId, String tenantId, Attr
 				if (t.rowCount() == 0) {
 					return Uni.createFrom().item(new HashMap<String, Object>());
 				}
+				Map<String,Object> result = t.iterator().next().getJsonObject(0).getMap();
+				if(attrsQuery!=null&& !attrsQuery.getAttrs().isEmpty() && result.size()<=4){
```

Comment:
Why are you adding this condition "result.size()<=4"

Reply:
id, type, createdat, modifiedat
if attrs query is not empty and still we get result size 4 or less then it should be "not found"